### PR TITLE
feat: tooltip fix for small screen

### LINKED ISSUE #1175: Nav icons hover on smaller screen size
### Overview

As a user, when I am viewing the web app on a smaller screen I want to be able to see what the nav icons are.  Currently, on smaller screens the icons become to small and are not readable. 

The nav icons should have a hover state on smaller screens.

### Reference

Bounty hunters might find this helpful

```
 <Tooltip label={label} isDisabled={isLargerThan2xl || !isCompact} placement='right'>
        <Button
          width='full'
          justifyContent='flex-start'
          variant='ghost'
          isActive={href ? active : false}
          minWidth={isCompact ? 'auto' : 10}
          iconSpacing={isLargerThan2xl ? 4 : isCompact ? 0 : 4}
          ref={ref}
          {...rest}
        >
          <Box display={{ base: isCompact ? 'none' : 'flex', '2xl': 'block' }}>{label}</Box>
        </Button>
      </Tooltip>
```

### Acceptance Criteria

Nav icons have a tooltip on smaller desktop screens and are more easily readable, see comment from @reallybeard below.

### Need By Date

n/a

### Screenshots/Mockups

Screen_Shot_2022-03-14_at_1 08 11_PM

### Ownership

- [X] If my bounty needs engineering or needs product I have added the respective labels on the right
- [X] As the sponsor of this bounty I will review the changes in a preview environment (ops/product) or review the PR (engineering)

### Bounty Hunters

- [Join our discord](https://discord.gg/shapeshift)
- Include an expected timeline for you to complete work in the work plan when you apply for this bounty!
- Please refer to [this link](https://shapeshiftdao.gitbook.io/getting-started/how-to-contribute/bounties#resources-for-new-contributors-and-bounty-hunters) for some basic info
- Please ***do not*** start work on this issue until you are approved in Gitcoin.

### PR DESCRIPTION
**Pull request by etherean06**
## Description

I've wrapped the Button component in a tool tip for small screens
## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)
closes #1175

If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes.

## Risk
Not risky at all, just a simple UI change with a Chakra UI component.

## Testing

If your testing steps are technical, outline steps for an engineer to verify.

If your testing steps are functional, please provide a full guide with any features requiring special attention for operations to test your branch in the preview environment.

## Screenshots (if applicable)
Screenshot 2022-03-19 at 19 16 01
Screenshot 2022-03-21 at 16 30 28
Screenshot 2022-03-21 at 16 30 17
Screenshot 2022-03-21 at 16 30 03
Screenshot 2022-03-21 at 16 29 49
Screenshot 2022-03-21 at 16 29 02

### CODE REVIEW

**Comment by NeOMakinG** [comment]
@etherean06 Be careful about the way you complete the PR template, some parts are not completed and checkboxes are wrong because of a space

File: src/components/Layout/Header/NavBar/MainNavLink.tsx [hunk 1]
```diff
@@ -30,6 +31,7 @@ export const MainNavLink = memo(
       >
         <Box display={{ base: isCompact ? 'none' : 'flex', '2xl': 'block' }}>{label}</Box>
       </Button>
+    </Tooltip>
```
Comment (NeOMakinG) [inline src/components/Layout/Header/NavBar/MainNavLink.tsx:34]:
```suggestion
      </Tooltip>
```

File: src/components/Layout/Header/NavBar/MainNavLink.tsx [hunk 2]
```diff
@@ -30,6 +31,7 @@ export const MainNavLink = memo(
       >
         <Box display={{ base: isCompact ? 'none' : 'flex', '2xl': 'block' }}>{label}</Box>
       </Button>
```
Comment (NeOMakinG) [inline src/components/Layout/Header/NavBar/MainNavLink.tsx:33]: Should be indented too, I think ESLint will throw it, with prettier

**Comment by etherean06** [comment]
@NeOMakinG My bad it was my first take

File: src/components/Layout/Header/NavBar/MainNavLink.tsx (hunk 2 above)
Comment (etherean06) [reply src/components/Layout/Header/NavBar/MainNavLink.tsx]: I modified the linting with ESLint, i think it is correct now

**Comment by NeOMakinG** [comment]
> @NeOMakinG My bad it was my first take

The `Issue` and `Risk` part is missing, adding `#<issuenumber>` is important because it will close the issue when your PR is merged

**Comment by gomesalexandre** [comment]
@etherean06 I tested it and it looks good, but could you please post screenshots of different viewports (both those where the tooltip is visible and those where it is hidden)?

**Comment by etherean06** [comment]
> @etherean06 I tested it and it looks good, but could you please post screenshots of different viewports (both those where the tooltip is visible and those where it is hidden)?

@gomesalexandre I've added a few more screenshots with different views :)

**Comment by etherean06** [comment]
> @reallybeard if you're happy with the breakpoints, I'll accept it!
> 
> @etherean06 Could you make some interactive rebase/squash/amends or whichever you prefer and rename the commits to follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
> 
> Title should follow this convention as well or CI will be very very sad.

I think i messed up the tree with my rebase, should i make another PR or is there a way to fix it?

**Comment by 0xdef1cafe** [comment]
> > @reallybeard if you're happy with the breakpoints, I'll accept it!
> > @etherean06 Could you make some interactive rebase/squash/amends or whichever you prefer and rename the commits to follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
> > Title should follow this convention as well or CI will be very very sad.
> 
> I think i messed up the tree with my rebase, should i make another PR or is there a way to fix it?

we can't help with git 101 stuff, sorry.

**Review by reallybeard — APPROVED** [review]
LGTM

**Comment by 0xdef1cafe** [comment]
also please include testing steps, e.g.

on x screen size a tooltip should appear for menu items, on y screen size it should not etc.

**Comment by gomesalexandre** [comment]
@etherean06 Please reset from a clean state off develop and cherry-pick https://github.com/shapeshift/web/pull/1228/commits/9911832ae60a4a5c36f4d1fdfb60e309fcd1cf95 and force push here - I can reopen this then.
I can't push in your branch since it's in your own namespace.

The testing steps mentioned by @0xdef1cafe are also not included.

I want to emphasize that this is a lot of back-and-forth work from 3 contributors already that have picked on things that are clearly in the GitHub PR template and repo READMEs  (PR titles, fields that weren't filled in the description, lint not passing etc).
This is on top of not being accepted to work on the bounty initially, which was also clearly stated there: `Please do not start work on this issue until you are approved in Gitcoin.`

That's a lot of developer power for a simple copy and paste work from @reallybeard's example.
Unless you're able to push a clean diff and have a clean PR that we're ready to merged without another round of changes requested, we're going to keep this close, sorry.

**Pull request closed without merging**